Allow JWT authorization options to support multiple issuers (#6887)
apollographql/router: docs/source/routing/security/jwt.mdx

- `url`: **required** URL from which the JWKS file will be read. Must be a valid URL.
  - **If you use a third-party IdP,** consult its documentation to determine its JWKS URL.
  - **If you use your own custom IdP,** you need to make its JWKS available at a router-accessible URL if you haven't already. For more information, see [Creating your own JWKS](#creating-your-own-jwks-advanced).

> - `issuers`: **optional** list of issuers accepted, that will be compared to the `iss` claim in the JWT if present. If one does not match, the request will be rejected.

Reviewer: ```suggestion
- `issuers`: **optional** list of issuers accepted, that will be compared to the `iss` claim in the JWT if present. If none match, the request will be rejected.
```

Author: Thank you for catching that!
